### PR TITLE
CXF-8079 Update to introduce the wsat transaction example

### PR DESCRIPTION
**Pull request by zhfeng**
https://issues.apache.org/jira/browse/CXF-8079

### CODE REVIEW

File: distribution/src/main/release/samples/ws_transaction/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+================
+It can wrap the local transaction and create a bridge between the JTA and WSAT transaction. 
+From the client side, we use the *JaxWSTxOutboundBridgeHandler* to create a mapping of the JTA and Subordinate WSAT
+also the *EnabledWSTXHandler* to propagate the WSAT transaction in the SOAP message headers.
+
+WSAT->JTA bridge in the server side
+================
+From the server side, we use the *JaxWSSubordinateHeaderContextProcessor* to import the Subordinate WSAT transaction from the outside
+and the *JaxWSHeaderContextProcessor* to resume the WSAT transaction and the *OptionalJaxWSTxInboundBridgeHandler* to create the bridge
+the WSAT and JTA.
```
Comment (dkulp) [inline distribution/src/main/release/samples/ws_transaction/README.md]: I think we should add a note to the README to mention that this example uses dependencies that are licensed under the LGPL and thus may have additional restrictions beyond the Apache License.    Explicitly listing the transaction manager (and hibernate) as having the potentially problematic license in the READ is likely good enough.    Users can then decide to use or not use the source in this example.

File: distribution/src/main/release/samples/ws_transaction/client/pom.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (dkulp) [inline distribution/src/main/release/samples/ws_transaction/client/pom.xml]: The apache license header needs to go in here.

**Comment by zhfeng** [comment]
Thanks @dkulp for reviewing and I will make changes as your suggestion.

File: distribution/src/main/release/samples/ws_transaction/client/pom.xml (hunk 2 above)
Comment (zhfeng) [reply distribution/src/main/release/samples/ws_transaction/client/pom.xml]: yeah, I added it.

**Comment by zhfeng** [comment]
@dkulp can you take a look at this PR and is there anything I have to do with the changes ?
Thanks !

**Pull request merged**